src/bookshelf/rebar.config

```diff
@@ -22,7 +22,7 @@
          {git, "git://github.com/mochi/mochiweb", {tag, "v2.12.2"}}},
         %% TODO 2017/02/09 sr: upgrade -- hex doesn't have that old version
         {erlware_commons, ".*",
-         {git, "https://github.com/erlware/erlware_commons", {tag, "v0.11.1"}}},
+         {git, "https://github.com/erlware/erlware_commons", {tag, "0.22.0"}}},
```

Comment:
I think this one has been stable enough to use master? We still get pinned when we `rebar3 upgrade`